### PR TITLE
pt-br translation of mine to support network PSA

### PR DESCRIPTION
**Pull request by rating89us**

### CODE REVIEW

**Comment by lh1008** [comment]
Thank you @rating89us :+1: 

Pinging @botoserelepe or @netrik182  for review. Please let me know who will do the review to add you to the taiga assigned tasks :).

**Comment by botoserelepe** [comment]
Hey, @lh1008  
I will do this review

**Comment by lh1008** [comment]
@botoserelepe thank you, @rating89us is this ready for review?

**Comment by rating89us** [comment]
ready

File: monero-outreach-docs/translations/pt-br/PSA/files/mining/mine_support_network_pt-br.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Minere Monero para apoiar a rede
+
+Os mineradores do Monero usam hardwares de computação para executar uma variante do algoritmo CryptoNight, competindo por uma recompensa de mineração que é distribuída cada vez que um novo bloco é adicionado à Blockchain. A mineração do Monero é conhecida por sua resistência aos ASICs (circuitos integrados para aplicações específicas). Isso significa que ela é mais acessível e amigável à mineração utilizando processadores comuns (CPU) e placas de vídeo (GPU), no entanto, ela não consegue impedir que indivíduos e empresas criem os poderosos ASICs e FPGAs. Em função disso, os desenvolvedores do Monero fazem a cada 6 meses uma atualização de rede de rotina, com o objetivo de evitar que a mineração através de ASICs e FPGAs ganhe controle da maior parte do hashrate da rede.
```
Comment (botoserelepe) [inline monero-outreach-docs/translations/pt-br/PSA/files/mining/mine_support_network_pt-br.md]: I think the text will be more fluent if you change the order of this sentence: "Em função disso, os desenvolvedores de Monero fazem uma atualização de rede de rotina a cada 6 meses, [...]"

File: monero-outreach-docs/translations/pt-br/PSA/files/mining/mine_support_network_pt-br.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+2.	**Mineração em Pool-** Essa é uma das maneiras mais comuns de se minerar Monero. Neste tipo de mineração você se associa a outros usuários para tentar descobrir juntos um bloco. Nesta modalidade os pagamentos tendem a ser muito mais regulares em comparação à mineração solo, mas eles são menores, pois você está dividindo a recompensa do bloco com todas as outras pessoas que ajudaram a minerá-lo. Leia mais em ([minexmr.com/#getting_started](https://minexmr.com/#getting_started)) e ([minexmr.com/pools.html](https://minexmr.com/pools.html)).
```
Comment (botoserelepe) [inline monero-outreach-docs/translations/pt-br/PSA/files/mining/mine_support_network_pt-br.md]: Use the plural in the second line "para tentar**em** descobrir juntos"

**Pull request merged**